release-notes/3.1/releases.json

```
@@ -1,13 +1,388 @@
 {
   "channel-version": "3.1",
-  "latest-release": "3.1.24",
-  "latest-release-date": "2022-04-12",
-  "latest-runtime": "3.1.24",
-  "latest-sdk": "3.1.418",
-  "support-phase": "lts",
+  "latest-release": "3.1.25",
+  "latest-release-date": "2022-05-10",
+  "latest-runtime": "3.1.25",
+  "latest-sdk": "3.1.419",
+  "support-phase": "LTS",
```

I wouldn't change this

```suggestion
  "support-phase": "lts",
```